packages/react-admin/src/mui/input/AutocompleteInput.js

```diff
@@ -91,65 +92,70 @@ const styles = theme => ({
  */
 export class AutocompleteInput extends React.Component {
     state = {
-        firstLoad: true,
+        dirty: false,
+        inputValue: null,
         searchText: '',
+        selectedItem: null,
```

Comment:
Please remove

Reply:
Euh no? This is a state variable, not a prop. It caches the selectedItem.